docs/configuration.md

```diff
@@ -82,6 +82,15 @@ emails.
 -   **Default value:** `("Budget manager", "admin@email.com")`
 -   **Production value:** Any tuple you want.
 
+## SHOW_ADMIN_EMAIL
+
+A boolean that determines whether the admin email (`MAIL_DEFAULT_SENDER`) is
+shown in error messages.
```

Comment:
You could also add the fact that if `MAIL_DEFAULT_SENDER` is not set, no email is shown.